user-interface/3002-PROP-propose.md

```diff
@@ -1 +1,60 @@
-# Propose
\ No newline at end of file
+# Propose
+
+Background: [Governance spec](../protocol/0028-GOVE-governance.md)
+and [docs](https://docs.vega.xyz/docs/mainnet/concepts/vega-protocol#governance).
+
+## Associate tokens
+
+When looking to propose something, I...
+
+- **must** see see a link to detailed explanation of Governance in the docs (<a name="3002-PROP-001" href="#3002-PROP-001">3002-PROP-001</a>)
+- **must** see that the governance token is an ethereum ERC20 token and needs to attributed (or [associated](1004-ASSO-associate.md)) to a Vega wallet for use on Vega (<a name="3002-PROP-002" href="#3002-PROP-002">3002-PROP-002</a>)
+- **must** see that I need some associated tokens, and a link to [associate](1004-ASSO-associate.md) (<a name="3002-PROP-003" href="#3002-PROP-003">3002-PROP-003</a>)
+
+... so I have a sufficient vote weight to propose.
+
+## Select proposal type
+
+When making a proposal, I...
+
+- **must** select a proposal type (<a name="3002-PROP-005" href="#3002-PROP-005">3002-PROP-005</a>)
+
+...so I get the appropriate form and information about rules for that type of proposal. e.g. min enactment and vote duration.
+
+## Populate a proposal form
+
+When making a proposal, I...
+
+- **must** input a rationale title and description (<a name="3002-PROP-006" href="#3002-PROP-006">3002-PROP-006</a>)
```

Comment:
must see the minimum tokens required to submit this type of proposal or amount of tokens for spam protection, whichever is greater?

Reply:
resolved in https://github.com/vegaprotocol/specs/pull/1302/commits/90f271efc59d9036bb3ef7d90e81fe168414d4fa ?